src/bin/hana_inst.sh

```diff
@@ -254,23 +255,12 @@ hana_lcm_workflow()
      rc=1
      return $rc
    fi
-   case $A_SAPMDC in
-     no)
-       echo -e "db_mode=\n"  > ${MEDIA_TARGET}/hana_mdc.conf
-     ;;
-     low)
-       echo -e "db_mode=multidb\ndb_isolation=low\n"  > ${MEDIA_TARGET}/hana_mdc.conf
-     ;;
-     high)
-       echo -e "db_mode=multidb\ndb_isolation=high\n"  > ${MEDIA_TARGET}/hana_mdc.conf
-     ;;
-   esac
    cd "${HDBLCMDIR}"
    TOIGNORE="check_signature_file"
    if [ -e /root/hana-install-ignore ]; then
      TOIGNORE=$(cat /root/hana-install-ignore)
    fi
-   if [ -e ${MEDIA_TARGET}/hana_mdc.conf ]; then
+   if [ ${XS_ROUTING_MODE} == "ports" ]; then
```

Comment:
same here. For SP6 the condition was changed to handle also empty variable cases and here it is not case.